odxtools/additionalaudience.py

```diff
@@ -16,9 +19,10 @@ class AdditionalAudience(IdentifiableElement):
     """
 
     @staticmethod
-    def from_et(et_element: ElementTree.Element,
-                doc_frags: List[OdxDocFragment]) -> "AdditionalAudience":
-        kwargs = dataclass_fields_asdict(IdentifiableElement.from_et(et_element, doc_frags))
+    def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment],
+                version: "Version") -> "AdditionalAudience":
```

Comment:
To make it more future proof, and avoid adding new arguments each time, maybe provide a "context" object, that has version and doc_frags as properties